docs/content/what-is-asynchronous-activity-completion.md

```diff
@@ -0,0 +1,13 @@
+---
+id: what-is-asynchronous-activity-completion
+title: What is Asynchronous Activity Completion?
+description: Asynchronous Activity Completion is when the final result of a computation, started by an Activity, is provided to the Temporal System from an external system.
```

Comment:
Should "...Completion is when..." be "...Completion occurs when..."? (Both here and in first sentence of body.)